ui/src/components/app-form/app-form.tsx

```diff
@@ -224,14 +219,14 @@ export const AppForm = ({ id }: AppFormProps): React.ReactElement => {
           jhub_app: true,
           name: currentServerName || displayName,
           display_name: displayName,
-          description: description || '',
-          framework,
-          thumbnail: thumbnail || '',
-          filepath: filepath || '',
-          conda_env: conda_env || '',
+          description: data.description || '',
+          framework: data.framework,
+          thumbnail: data.thumbnail || '',
+          filepath: data.filepath || '',
+          conda_env: data.conda_env || '',
           env: getFriendlyEnvironmentVariables(variables),
-          custom_command: custom_command || '',
-          profile: profile || '',
+          custom_command: data.custom_command || '',
+          profile: data.profile || '',
```

Comment:
Same comment as above

Reply:
Not sure where that's coming from. This is what I have:
<img width="464" alt="image" src="https://github.com/nebari-dev/jhub-apps/assets/885376/ddacc927-9860-481c-a1a2-a2aec49cd491">

Reply:
I'm referring to using the data object, as opposed to just the individual values.